### PR TITLE
doc: edit scrub and LTS straw man refinement

### PR DESCRIPTION
**Pull request by jasnell**
additional edit scrub and refinement on the LTS straw man model

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -169,31 +179,58 @@ The LTS WG is expected to establish a regular and predictable cadence of stable
...
+
+Once the `-rc.1` tag is cut at a particular point, the master stream goes into "Feature Freeze" until the release branch is cut. New features landed before the `-rc.1` tag make it into the LTS Release, those that do not must wait to land until after the Feature Freeze is lifted.
```
Comment (Fishrock123) [inline README.md:186]: Seems unnecessary. Cut a branch then tag it in one go.

**Comment by Fishrock123** [comment]
The language here is a little confusing, feature freezes should only happen for semver-minor stuff when a  patch release has been proposed and is about to go out.

LTS releases should not have to freeze master, ever.

**Comment by Fishrock123** [comment]
Also, I think there was some discussion that LTS shouldn't bump minors. If so, why not just continue a branch on regular patch-level?

**Comment by jasnell** [comment]
Noted. I'm going to hold off further major edits until after the call today and bundle everything up. The LTS discussion is a strawman at this point so nothing there is anywhere close to being complete :-)

**Comment by jasnell** [comment]
Btw @Fishrock123 ... thank you very much for all the feedback this week. It's been extremely helpful.

**Pull request merged**